client/state/plugins/premium/test/selectors.js

```diff
@@ -30,144 +36,134 @@ const state = deepFreeze( {
-			assert.equal( selectors.getPluginsForSite( state, 'start.site' )[ 1 ].slug, 'akismet' );
-			assert.equal( selectors.getPluginsForSite( state, 'start.site' )[ 2 ].slug, 'polldaddy' );
+			expect( getPluginsForSite( state, 'start.site' ) ).toHaveLength( 3 );
+			expect( getPluginsForSite( state, 'start.site' )[ 0 ].slug ).toBe( 'vaultpress' );
+			expect( getPluginsForSite( state, 'start.site' )[ 1 ].slug ).toBe( 'akismet' );
+			expect( getPluginsForSite( state, 'start.site' )[ 2 ].slug ).toBe( 'polldaddy' );
 		} );
 	} );
 
 	describe( 'getActivePlugin', () => {
 		test( 'Should get `false` if the requested site is not in the current state', () => {
-			assert.equal( selectors.getActivePlugin( state, 'no.site' ), false );
+			expect( getActivePlugin( state, 'no.site' ) ).toBe( false );
 		} );
 
 		test( 'Should get `false` if no plugins on the requested site are currently being installed', () => {
-			assert.equal( selectors.getActivePlugin( state, 'start.site' ), false );
+			expect( getActivePlugin( state, 'start.site' ) ).toBe( false );
 		} );
 
 		test( 'Should get `false` if all plugins on the requested site are finished installing', () => {
-			assert.equal( selectors.getActivePlugin( state, 'finished.site' ), false );
+			expect( getActivePlugin( state, 'finished.site' ) ).toBe( false );
 		} );
 
 		test( 'Should get `akismet` if akismet is the currently being installed on the requested site', () => {
-			assert.equal( selectors.getActivePlugin( state, 'installing.site' ).slug, 'akismet' );
+			expect( getActivePlugin( state, 'installing.site' ).slug ).toBe( 'akismet' );
 		} );
 	} );
 
 	describe( 'getNextPlugin', () => {
 		test( 'Should get `false` if the requested site is not in the current state', () => {
-			assert.equal( selectors.getNextPlugin( state, 'no.site' ), false );
+			expect( getNextPlugin( state, 'no.site' ) ).toBe( false );
 		} );
 
 		test( "Should get the first plugin in the list if the requested site hasn't started yet", () => {
-			assert.equal( selectors.getNextPlugin( state, 'start.site' ).slug, 'vaultpress' );
+			expect( getNextPlugin( state, 'start.site' ).slug ).toBe( 'vaultpress' );
 		} );
 
 		test( 'Should get `polldaddy`, next in the list, if the requested site is installing akismet', () => {
-			assert.equal( selectors.getNextPlugin( state, 'installing.site' ).slug, 'polldaddy' );
+			expect( getNextPlugin( state, 'installing.site' ).slug ).toBe( 'polldaddy' );
 		} );
 
 		test( 'Should get `false`, if the requested site is finished installing all plugins', () => {
-			assert.equal( selectors.getNextPlugin( state, 'finished.site' ), false );
+			expect( getNextPlugin( state, 'finished.site' ) ).toBe( false );
+		} );
+	} );
+
+	describe( 'getPluginKeys', () => {
+		test( 'Should get an empty object if there are no plugins for this site', () => {
+			expect( getPluginKeys( state, 'no.site' ) ).toEqual( {} );
+		} );
+
+		test( 'Should get plugin keys if there are plugins for this site', () => {
```

Comment:
A "repeated calls returns same object" might be nice, but up to you.

Reply:
Good call - just added one in https://github.com/Automattic/wp-calypso/pull/31684/commits/ae588e79c43dd9e66ae0a89e512f585a5f47c3e1